apps/api/src/app/message-template/usecases/update-message-template/update-message-template.usecase.ts

```diff
@@ -1,3 +1,4 @@
+import { AnyKeys } from 'mongoose';
```

Comment:
Domain application should be agnostic of the ORM. We mentioned this in the previous PR but for a different file. Just a reminder. 🙏🏻